.github/actions/deploy-infra/action.yml

```diff
@@ -41,6 +41,9 @@ inputs:
   graasp-db-gatekeeper-key-name:
     description: "Gatekeeper key name for auth"
     required: true
+  graasper-id:
+    description: "Graasper user id"
```

Comment:
should we add an indication that it is used for fetching its collections?

Reply:
Yes I can add this info